core/src/main/java/com/sequenceiq/cloudbreak/service/rdsconfig/ProfilerAdminConfigProvider.java

```diff
@@ -0,0 +1,67 @@
+package com.sequenceiq.cloudbreak.service.rdsconfig;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.base.DatabaseType;
+import com.sequenceiq.cloudbreak.blueprint.AmbariBlueprintProcessorFactory;
+import com.sequenceiq.cloudbreak.blueprint.AmbariBlueprintTextProcessor;
+import com.sequenceiq.cloudbreak.domain.Blueprint;
+import com.sequenceiq.cloudbreak.service.blueprint.BlueprintService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+@Component
+public class ProfilerAdminConfigProvider extends AbstractRdsConfigProvider {
+    private static final String PILLAR_KEY = "profiler_admin";
+
+    @Value("${cb.profiler.admin.database.user:profileragent}")
+    private String profilerAdminDbUser;
+
+    @Value("${cb.profiler.admin.database.db:profileragent}")
+    private String profilerAdminDb;
+
+    @Value("${cb.profiler.admin.database.port:5432}")
+    private String profilerAdminDbPort;
+
+    @Inject
+    private AmbariBlueprintProcessorFactory ambariBlueprintProcessorFactory;
```

Comment:
This should rather be `CmTemplateProcessorFactory` as we are dealing with CmTemplates.

Reply:
I kept the same way as other services like Oozie, Hue, etc. But well if `CmTemplateProcessorFactory` will be used in the future. I will change this. I hope below API should work.
```
CmTemplateProcessor cmTemplateProcessor = cmTemplateProcessorFactory.get(blueprintText);
return cmTemplateProcessor.isComponentExistsInHostGroup("PROFILER_METRICS_AGENT", "master");
```